sql/database.sql

```diff
@@ -7899,6 +7899,7 @@ INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES (
 
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('lists', 'page_validation', 'Page Validation', 298);
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'add_edit_issue#theform', '/interface/patient_file/summary/add_edit_issue.php', 10, '{form_title:{presence: true}}', 0);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'encounters#new_encounter', '/interface/forms/newpatient/new.php', 20, '{pc_catid:{exclusion: ["_blank"]}}', 0);
```

Comment:
that is interesting. that also mean there would of needed to be an entry for view.php if was using validateUsingPageRules which would not of been a big deal (note this is not an issue here, though, since bypassing validateUsingPageRules).